docker/dev.yml

```diff
@@ -5,6 +5,8 @@ x-backend-conf: &backend-conf
   build:
     context: .
     dockerfile: Dockerfile
+    cache_from:
+      - ghcr.io/openfoodfacts/openfoodfacts-server/backend:buildcache
```

Comment:
Aren't you trading local CPU time to network bandwith ? Because most of students that have problem running the project have bandwith problem rather than CPU problems…

Reply:
In my local builds, I had issues downloading the apt packages way more often than downloading container images. My hope is that it could help by downloading less in total. I have to admit that haven't measured that, yet. Also, I don't know if downloading a single layer vs. multiple packages is usually better with a flaky connection 🤔 maybe someone with issues building the image from main could test this to compare?